src/connection/host_lookup.c

```diff
@@ -303,37 +304,35 @@ load_hosts_file ()
 		  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_UHOST_INVALID_FORMAT, 4, "IP address", token, line_num,
 			  USER_HOSTS_FILE);
 
-		  user_host_Map.clear ();
-		  fclose (fp);
-		  goto load_fail_phase;
+		  continue;
 		}
 	      else
 		{
 		  strncpy (ipaddr, token, IPADDR_LEN);
 		  ipaddr[IPADDR_LEN - 1] = '\0';
-
 		  hostent_flag = INSERT_HOSTNAME;
 		}
 	    }
 	  else
 	    {
-	      if (strlen (token) > HOSTNAME_LEN - 1)
+	      str_len = strlen (temp_token);
+	      if (str_len > HOSTNAME_LEN - 1)
 		{
-		  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_UHOST_HOST_NAME_TOO_LONG, 3, token, line_num,
+		  continue;
+		}
+	      else if (ip_format_check (temp_token) == true)
+		{
+		  er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_UHOST_INVALID_FORMAT, 4, "Hostname", token, line_num,
 			  USER_HOSTS_FILE);
 
-		  user_host_Map.clear ();
-		  fclose (fp);
-		  goto load_fail_phase;
+		  continue;
 		}
-	      else if (ip_format_check (temp_token) == true)
+	      else if (hostname_format_check (temp_token, str_len) == false)
```

Comment:
what about change function name as (is_valid_hostname): _trivial_
else if (**is_valid_hostname** (temp_token, str_len) == false)

Reply:
If then, I will change "ip_format_check" to "is_valid_ip" either.